src/main/java/com/ays/backend/user/model/EType.java

```diff
@@ -0,0 +1,6 @@
+package com.ays.backend.user.model;
+
+public enum EType {
+    DEVICE_1,
```

Comment:
Please add values to these enums, as you'll have indices instead, when you want to get the value of it. Please be more specific about these enumerations, as these doesn't make sense.